Userland/Applications/Terminal/main.cpp

```diff
@@ -277,6 +277,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     window->set_background_color(Color::Black);
     window->set_double_buffering_enabled(false);
 
+    // Resizing the terminal to a very small size will provoke a crash in LibLine, this a safeguard.
+    // This set the minimal viewable size to 16x2.
+    window->set_minimum_size(136, 36);
```

Comment:
This is not an appropriate fix. If there's a crash in LibLine, we should be looking at LibLine, not adding random hacks to Terminal.

Reply:
I thought that it was the only way to trigger the bug, and in my opinion there is no point to support a null-sized Shell.
It turns out (only for me), that the bug is much bigger than that.
And, I totally agree that it is not a proper fix, you can close the PR if you want.